addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -878,14 +874,15 @@ action = act
     <record id="hr_applicant_action_from_department" model="ir.actions.act_window">
         <field name="name">New Applications</field>
         <field name="res_model">hr.applicant</field>
-        <field name="view_mode">kanban,tree,form,graph,calendar,pivot</field>
+        <field name="view_mode">kanban,tree,form,graph,calendar,pivot,list</field>
```

Comment:
view_mode "list" ? 
Don't even know what that is.
Looks like an old compatibility layer for "tree".

Why did you add that?

Reply:
I did it by accident , i did removed the entire kanban to test something when , I did put back i did the error maybe an copy paste of another kanban